python/ray/tune/experiment.py

```diff
@@ -105,6 +106,11 @@ def from_json(cls, name, spec):
         if "run" not in spec:
             raise TuneError("No trainable specified!")
 
+        if "repeat" in spec:
+            warnings.warn("repeat is deprecated, use total_samples instead")
```

Comment:
instead of `warnings`, can you do the following:

```
raise DeprecationWarning("The parameter `repeat` is deprecated; converting to `num_samples`. 
						 `repeat` will be removed in future versions of Ray.")```

Reply:
this would also need you to remove the import above.